src/bundle/ui-dev/src/modules/multi-file-upload/components/upload-list/upload.list.component.js

```diff
@@ -123,12 +100,16 @@ export default class UploadListComponent extends Component {
 
     render() {
         const { itemsToUpload } = this.props;
-        const { items } = this.state;
+        const { items, erroredItems } = this.state;
+        const uploadListClassName = createCssClassNames({
+            'c-upload-list': true,
+        });
 
         return (
-            <div className="c-upload-list">
+            <div className={uploadListClassName}>
```

Comment:
```suggestion
            <div className="c-upload-list">
```
and remove 104-106? you don't have conditional class there